### PR TITLE
 added WG etcd operator community meetings to community section

### PR DESCRIPTION
**Pull request by danish9039**
resolves issue #896

WG etcd operator community meetings to community section

### CODE REVIEW

File: content/en/community/_index.md [hunk 1]
```diff
@@ -79,6 +79,17 @@ started contributing.
 For phone-in information, the date of the next meeting, and minutes from past
 meetings, see [etcd community meeting doc][community-meeting-doc].
 
+## WG etcd Operator Meetings
```
Comment (jmhbnz) [inline content/en/community/_index.md:82]:
```suggestion
## Operator working group
```

File: content/en/community/_index.md [hunk 2]
```diff
@@ -79,6 +79,17 @@ started contributing.
...
+
+Join the WG etcd Operator community for discussions on the development and management of the etcd operator. These meetings are held biweekly and are open to all community members who wish to contribute or stay informed about the project.
```
Comment (jmhbnz) [inline content/en/community/_index.md:84]: Let's include a link here to the working group 
```suggestion
Join the [etcd operator working group](https://github.com/kubernetes/community/tree/master/wg-etcd-operator) for discussions on the development and management of the etcd operator. These meetings are held biweekly and are open to all community members who wish to contribute or stay informed about the project.
```

File: content/en/community/_index.md [hunk 3]
```diff
@@ -79,6 +79,17 @@ started contributing.
...
+
+**Meeting Schedule:**  
+- **Biweekly** on **Tuesdays at 11 a.m. PT**.
+
+**Zoom Details:**  
+The Zoom details are the same as the Robustness and Community/Triage meetings.
```
Comment (jmhbnz) [inline content/en/community/_index.md:90]: We could also add a link to the meeting notes doc
```suggestion
The Zoom details are the same as the Robustness and Community/Triage meetings. Meeting notes are available [here](https://docs.google.com/document/d/1ey4zTTRvtCVJJP2vjF95VjG-sAKlNTcqB2HdmC18Lfc/edit?usp=sharing).
```

**Review by jmhbnz — COMMENTED** [review]
Thanks for updating this @danish9039, a few suggestions below 🙏🏻

File: content/en/community/_index.md [hunk 4]
```diff
@@ -79,6 +79,18 @@ started contributing.
...
+Join the [etcd operator working group](https://github.com/kubernetes/community/tree/master/wg-etcd-operator) for discussions on the development and management of the etcd operator. These meetings are held biweekly and are open to all community members who wish to contribute or stay informed about the project.
+
+**Meeting Schedule:**
+
+- **Biweekly** on **Tuesdays at 11 a.m. PT**.
+
+**Zoom Details:**  
+The Zoom details are the same as the Robustness and Community/Triage meetings. Meeting notes are available [here](https://docs.google.com/document/d/1ey4zTTRvtCVJJP2vjF95VjG-sAKlNTcqB2HdmC18Lfc/edit?usp=sharing).
+
+
```
[the remark this answers is not in the document]
Comment (danish9039) [reply content/en/community/_index.md:93]: yeah sure

**Review by ivanvc — APPROVED** [review]
LGTM. I think we just need a squash of the commits.

**Review by spzala — COMMENTED** [review]
lgtm but a nit - In both the sections on this same page `Community meetings & Issue triage` and `Robustness tests` the meeting time is mentioned as below:
... AM [Pacific Time](https://www.timeanddate.com/time/zones/pt)

We may want to use the same format instead of using ` ... a.m. PT.`

**Review by jmhbnz — APPROVED** [review]
LGTM - Happy for the AM formatting to be addressed as a follow-up.

**Pull request merged**